opencti-platform/opencti-front/src/private/components/data/ToolBar.jsx

```diff
@@ -1346,13 +1346,17 @@ class ToolBar extends Component {
     const notEnrichableTypes = ['Label', 'Vocabulary', 'Case-Template', 'Task', 'DeleteOperation'];
     const isManualEnrichSelect = !selectAll && selectedTypes.length === 1;
     const isAllEnrichSelect = selectAll
-        && entityTypeFilterValues.length === 1
-        && entityTypeFilterValues[0] !== 'Stix-Cyber-Observable'
-        && entityTypeFilterValues[0] !== 'Stix-Domain-Object';
+      && entityTypeFilterValues.length === 1
+      && entityTypeFilterValues[0] !== 'Stix-Cyber-Observable'
+      && entityTypeFilterValues[0] !== 'Stix-Domain-Object';
     const enrichDisable = notEnrichableTypes.includes(selectedTypes[0])
-        || (entityTypeFilterValues.length === 1
-            && notEnrichableTypes.includes(entityTypeFilterValues[0]))
-        || (!isManualEnrichSelect && !isAllEnrichSelect);
+      || (entityTypeFilterValues.length === 1
+        && notEnrichableTypes.includes(entityTypeFilterValues[0]))
+      || (!isManualEnrichSelect && !isAllEnrichSelect);
+    // endregion
+    // region orgaSharing
+    const notShareableTypes = ['Label', 'Vocabulary', 'Case-Template', 'DeleteOperation'];
+    const isShareableType = !notShareableTypes.includes(selectedTypes[0]);
```

Comment:
shall we put this in a utility function somewhere?

Reply:
Yep, we will have to rework Toolbar someday